src/core/viz/expressions/ramp.js

```diff
@@ -12,30 +13,28 @@ import { cielabToSRGB, sRGBToCielab } from '../colorspaces';
 * @param {carto.expressions.palettes} palette - The color palette that is going to be used
 * @return {carto.expressions.Base}
 *
-* @example <caption> Display points with a different color depending on the `category` property. (We assume category has discrete values) </caption>
+* @example <caption>Display points with a different color depending on the `category` property. (We assume category has discrete values).</caption>
 * const s = carto.expressions;
 * const viz = new carto.Viz({
-*   color: s.ramp(s.prop('category'), s.palettes.PRISM),
+*   color: s.ramp(s.prop('category'), s.palettes.PRISM)
 * });
 *
-* @example <caption> Display points with a different color depending on the `speed` property. (We assume category has continuos numeric values)</caption>
```

Comment:
Why have you remove this example?
By the way: 👮 /continuos/continuous

Reply:
This example adds nothing new. The same as before but changing the column's name.